server/chat-plugins/thecafe.tsx

```diff
@@ -1,5 +1,6 @@
 import {FS} from '../../lib/fs';
 import {Utils} from '../../lib/utils';
+import preact from 'preact';
```

Comment:
You're just importing `preact` for the `preact.VNode` type? Probably that should be put on `Chat`?

Reply:
The imports are needed for every invocation of JSX; the compiler needs preact.h. We could maybe put that on Chat, sure? I'll try it.

Reply:
Done.

Reply:
Whoa, ok, it doesn't like that.

Reply:
Tests pass locally - i'm unsure what the failures are but it seems rooted in this?